scaffolder-templates/github-workflows/advanced-workflow/docs/GithubAdvancedDoc.md

```diff
@@ -0,0 +1,85 @@
+
+### Page 1: Workflow Software Project Configuration
+
+On this page the user will provide input for the following parameters:
+
+- Organization Name: The github organization name of the newly created workflow. This Organization needs to exist prior to running the template. 
+- Repository Name: The name of the github repository that will hold the source code for the to-be-created serverless workflow. 
+- Description: Description added to the README file
+- Workflow ID: A unique ID for the serverless workflow in SonataFlow. 
+- Infrastructure Workflow ID: Workflow ID, the unique identifier of the infrastructure worklow available in the environment
+- Owner: A drop down menu will reveal potential owners for this workflow
+- System: An entity from the catalog.
+
+### Page 2: Build Enviroment
+
+On this page, the user will choose a CI/CD method for building and monitoring their serverless worklow application.
+Upon Choosing 'Tekton with ArgoCD', the user will be prompted to enter some following input parameters.
+
+#### Repository Configuration
+
+- Workflow Namespace: Deployment namespace for workflow applications
+- GitOps Namespace: Deployment namespace for ArgoCD and Tekton resources
+
+#### PostreSQL Configuration:
+
+- PostgreSQL Secret Name: Name of the secret in which the PostgreSQL secrets are stored. Shall be in the same namespace as the workflow
+- PostgreSQL User key from secret: The key name in which the PostgreSQL *user* is stored
+- PostgreSQL Password key from secret: The key name in which the PostgreSQL *password* is stored
+- PostgreSQL K8s Service Name: Name of the service running the PostgreSQL instance
+- PostgreSQL K8s Service Namespace: Namespace of the service running the PostgreSQL instance
+- PostgreSQL Port: Port on which the PostgreSQL instance is running
+
+#### Quay Configuration: 
+
+- Quay Organization Name: The Quay Organization Name of the published workflow
+- Quay Repository Name: A name for the Quay Repository of the published workflow, either existing or to be created
+
+The user can choose an existing Quay repository for pushing the built images, or creating a new one.
+To create a new Quay repository, some additional configuration needs to be made: 
+
+1. The [following dynamic plugin](https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-quay) needs to be added to RHDH. 
+To add this plugin, the user can add a new package to the dynamic plugin CR in the rhdh-operator namespace on the cluster
+```
+- disabled: false
+  package: ./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-quay
+```
+2. Create or locate an Organization on Quay
+3. Create an OAuth Application in the organization
+4. Configure 'Create Repositories Access' and grant the OAuth app permission to create new repositories: ![image without spacing](assets/QuayOAuth.png)
+5. Generate an Access Token to use
+
+Upon choosing to create a new one, the user will be prompted for additional input parameters
```

Comment:
Same as for the CI, it would be nice to state the obvious: if the user choose to not create a new quay repository, it must exists (and maybe have some settings configured)